assets/src/edit-story/elements/video/karma/autoplay.js

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import { Fixture } from '../../../karma';
+import { getBackgroundElementId } from '../../../components/dropTargets/karma/background.karma';
+
+fdescribe('Autoplay video', () => {
+  let fixture;
+
+  beforeEach(async () => {
+    fixture = new Fixture();
+    await fixture.render();
+  });
+
+  afterEach(() => {
+    fixture.restore();
+  });
+
+  it('should render ok', () => {
+    expect(
+      fixture.container.querySelector('[data-testid="fullbleed"]')
+    ).toBeTruthy();
+  });
+
+  it('should autoplay on insert and on drop', async () => {
+    // TODO: Switch to role selector after they are merged
+    const videoFilter = fixture.screen.getByText('Video', {
+      exact: true,
+      selector: 'button',
+    });
+    await fixture.events.mouse.clickOn(videoFilter);
+    const videos = fixture.screen.getAllByTestId('mediaElement');
+    await fixture.events.mouse.clickOn(videos[0]);
+    const frames = fixture.screen.getAllByTestId('frameElement');
+    const videoFrame = frames[1];
+    const videoId = videoFrame.dataset.elementId;
+    const video1El = fixture.querySelector(`#video-${videoId}`);
+    expect(video1El.paused).toBe(false);
+
+    // TODO: After #1948 is merged we can check if video is paused during a drag
+    const safezone = fixture.screen.getAllByTestId('safezone')[0];
+    await fixture.events.mouse.seq(({ moveRel, down, up }) => [
+      moveRel(videoFrame, '50%', '50%', { steps: 12 }),
+      down(),
+      moveRel(safezone, 10, 10, { steps: 12 }),
+      up(),
+    ]);
+    const backgroundId = await getBackgroundElementId(fixture);
```

Comment:
This line will break this test because the fixture is rerendered so the video is no longer playing. I can see that behavior breaking other places so we need to watch out.

Reply:
Hm, this shouldn't be an issue since [this fix](https://github.com/google/web-stories-wp/pull/2261/files#diff-8ced9a717a92882ca8bd360f336ad948R310-R313), but perhaps that didn't fix it in all instances? Ping @dvoytenko

Reply:
Correct, it is now fixed 🎉